pac2go.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+function die() {
+  echo $*
+  exit 1
+}
+which 2goarray > /dev/null || go get github.com/jteeuwen/go-bindata/...
+which 2goarray > /dev/null || die "Please install 2goarray manually, then try again"
```

Comment:
That's a good reason.  2goarray looks cool and I think I'll start using it instead of go-bindata.